pkg/legacy/importer_test.go

```diff
@@ -1,50 +0,0 @@
-// Unless explicitly stated otherwise all files in this repository are licensed
-// under the Apache License Version 2.0.
-// This product includes software developed at Datadog (https://www.datadoghq.com/).
-// Copyright 2017 Datadog, Inc.
-
-package legacy
-
-import (
-	"fmt"
-	"testing"
-
-	"github.com/DataDog/datadog-agent/pkg/collector/py"
```

Comment:
This file doesn't seem to be related, bad merge?

Reply:
Yes, you're right - fixing.